views/terms.hbs

```diff
@@ -0,0 +1,108 @@
+<div class="container-fullwidth land-splash">
```

Comment:
classes like land-splash shouldn't be used here, they're only for the landing page. It includes undesired behavior like scroll jumping and a coloured background which makes it hard to read.

Reply:
Also is there a way to deal with navbar blocking the top of the container?